modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -1877,6 +1883,18 @@ private boolean inverseTcpConnectionFeatureIsSupported(ClusterNode node) {
         return new GridIoMessage(plc, topic, topicOrd, msg, ordered, timeout, skipOnTimeout);
     }
 
+    /**
+     * @return Version of security processor feature supported by cluster.
+     */
+    private IgniteFeatures currentSecurityProcSupport() {
+        if (allNodesSupports(ctx, ctx.discovery().allNodes(), IGNITE_SECURITY_PROCESSOR_V2))
```

Comment:
Lets eliminate double call of `ctx.discovery().allNodes()` method.

Reply:
fixed